### PR TITLE
DMD-832 Lock Menu Items

### PR DESCRIPTION
**Pull request by tomasz-ludek**
Hide wallet functions which pertain to user's personal settings and linked accounts when wallet is locked in order to avoid leaking sensitive data.

### CODE REVIEW

**Comment by HashEngineering** [comment]
This is an interesting approach.

Menu items are hidden if the app is locked.

**Comment by tomasz-ludek** [comment]
Rebased onto `master`

File: wallet/src/de/schildbach/wallet/ui/WalletActivity.java [hunk 1]
```diff
@@ -417,9 +417,31 @@ protected void error(final int messageResId, final Object... messageArgs) {
     public boolean onCreateOptionsMenu(final Menu menu) {
         getMenuInflater().inflate(R.menu.wallet_options, menu);
         super.onCreateOptionsMenu(menu);
+
+        getMenuInflater().inflate(R.menu.wallet_options, menu);
```
Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: This is a duplicate of what is two lines above.

Comment (tomasz-ludek) [reply wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: I fixed this issue and rebased onto master.

**Comment by HashEngineering** [comment]
This was merged manually here:  https://github.com/HashEngineering/dash-wallet/commit/bb980c2ff57ea2e3da0eb864a800432036d67f7f

**Pull request closed without merging**